### ISSUE TITLE
Disable prepared statements by default

### ISSUE THREAD

**Issue by grypyrg** [akopytov/sysbench#310]
(Related to issue #126 & #217)

Please seriously consider disabling prepared statements by default in `sysbench`:

* In the real world production environments, prepared statements are not frequently used (hardly ever)
* as sysbench creates multiple tables, and prepared statements have to be created per table/query/connection, sysbench has to create and manage A LOT of prepared statements. 

`sysbench` users often configure a large/insane amount of threads and/or tables to run benchmarks, which in turn they need the database server to support an insane amount of prepared statements and will run into errors. Nothing anybody would hit in the real world.

**Comment by akopytov** [comment]
@grypyrg I agree. Will disable it by default in the master branch (to be released as the next major version).

**Comment by grypyrg** [comment]
Thanks @akopytov for the quick response!
